<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>530</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>770</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Column</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>451</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="headerHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Search Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Search Select</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Text</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>TableView</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>760</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>Converter</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Load_JP">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>711</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 0, 150)</string>
      </property>
      <property name="text">
       <string>LOAD JAPAN FILE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cvt_KR">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 170, 0, 150)</string>
      </property>
      <property name="text">
       <string>CONVERT EXCEL KR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cvt_Txt">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 170, 255, 150)</string>
      </property>
      <property name="text">
       <string>CONVERT TXT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab4">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionConnect_To_DataBase"/>
    <addaction name="actionSelect_Table"/>
    <addaction name="actionLoad_Table_For_Update"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QMenu" name="menuSearch_Select">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Search Select</string>
     </property>
    </widget>
    <addaction name="menuSearch_Select"/>
    <addaction name="actionSearch_Text"/>
    <addaction name="actionSearch_Japan_Name"/>
    <addaction name="actionSearch_Korea_Name"/>
    <addaction name="actionCount_Histogram"/>
   </widget>
   <widget class="QMenu" name="menuTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Table</string>
    </property>
    <addaction name="actionCell_Marking"/>
    <addaction name="actionClearCell_Marking"/>
    <addaction name="actionText_Marking"/>
    <addaction name="actionClear_Text_Marking"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuSearch"/>
   <addaction name="menuTable"/>
  </widget>
  <action name="actionSearch_Text">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search Text</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionConnect_To_DataBase">
   <property name="text">
    <string>Connect To DataBase</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionCell_Marking">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cell Marking</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionClearCell_Marking">
   <property name="text">
    <string>Clear Cell Marking</string>
   </property>
  </action>
  <action name="actionText_Marking">
   <property name="text">
    <string>Text Marking</string>
   </property>
  </action>
  <action name="actionClear_Text_Marking">
   <property name="text">
    <string>Clear Text Marking</string>
   </property>
  </action>
  <action name="actionSearch_Japan_Name">
   <property name="text">
    <string>Search Japan Name</string>
   </property>
  </action>
  <action name="actionSelect_Table">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select Table</string>
   </property>
  </action>
  <action name="actionLoad_Table_For_Update">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Table For Update</string>
   </property>
  </action>
  <action name="actionSearch_Korea_Name">
   <property name="text">
    <string>Search Korea Name</string>
   </property>
  </action>
  <action name="actionCount_Histogram">
   <property name="text">
    <string>Count Histogram</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
